content/en/blog/drivers-add-new-distro.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Add prebuilt drivers for new distro
+date: 2023-03-21
+author: Federico Di Pierro
+slug: falco-prebuilt-drivers-new-distro
+---
+
+Hi everyone!  
+Today we are going to learn how to add support for a new distro prebuilt drivers.  
+There are multiple repositories involved with it, and while most of the time it should be a pretty simple job,  
+other times it can become really cumbersome.  
+That's why we are writing this guide of course!  
+
+The involved repositories will be:
+* [Kernel-crawler](https://github.com/falcosecurity/kernel-crawler)
+* [Driverkit](https://github.com/falcosecurity/driverkit)
+* [Test-infra](https://github.com/falcosecurity/test-infra)
+* [Falco](https://github.com/falcosecurity/falco)
+
+## Kernel-crawler
+
+> **NOTE**: this step is only needed if kernel-crawler does not already support the distro.
+
+Kernel-crawler is a python project that is capable of fetching all existing kernels by scraping multiple distros repositories.  
+It is ran weekly on Monday, by a [github action](https://github.com/falcosecurity/kernel-crawler/blob/main/.github/workflows/update-kernels.yml), to update the jsons for all supported architectures (at the moment of writing `x86_64` and `aarch64`) by opening a PR on same kernel-crawler against the `kernels` branch.  
```

Comment:
Maybe be more accurate by telling the exact schedule, we had questions about that.

Reply:
> It is ran weekly on Monday

How can i write it better? :D

Reply:
It runs every Monday at 6AM UTC